webapi/src/main/scala/org/knora/webapi/responders/v1/StandoffResponderV1.scala

```diff
@@ -435,27 +464,35 @@ class StandoffResponderV1 extends ResponderV1 {
       * @param userProfile the client that made the request.
       * @return a [[CreateStandoffResponseV1]]
       */
-    private def createStandoffV1(projectIri: IRI, resourceIri: IRI, propertyIRI: IRI, xml: String, userProfile: UserProfileV1, apiRequestID: UUID): Future[CreateStandoffResponseV1] = {
+    private def createStandoffV1(projectIri: IRI, resourceIri: IRI, propertyIRI: IRI, mappingIri: IRI, xml: String, userProfile: UserProfileV1, apiRequestID: UUID): Future[CreateStandoffResponseV1] = {
 
         val standoffUtil = new StandoffUtil()
 
-        val textWithStandoff: TextWithStandoff = standoffUtil.xml2TextWithStandoff(xml)
+        // FIXME: if the XML is not well formed, the error is not handled correctly
+        // FIXME: this should be done in StandoffUtil but the error is swallowed (Future issue?)
 
-        // TODO: get the mapping that was used when creating the standoff values
-        val mappingXMLtoStandoff: MappingXMLtoStandoff = getMapping("")
-
-        // collect standoff classes Iris from mapping
-        val standoffTagIris: Set[IRI] = mappingXMLtoStandoff.namespace.flatMap {
-            case (namespace: String, mapping: Map[String, XMLTag]) =>
-                mapping.map {
-                    case (tagname: String, tagItem: XMLTag) =>
-                        tagItem.mapping.standoffClassIri
-                }
-        }.toSet
+        val textWithStandoff: TextWithStandoff = try {
+            standoffUtil.xml2TextWithStandoff(xml)
```

Comment:
Parsing errors due to not well formed XML should be caught in StandoffUtil

@benjamingeer The errors were swallowed and then the future timed out